src/client/index.js

```diff
@@ -38,7 +38,9 @@ class Hammerhead {
         // Methods
         this.getOriginElementAttributes = CodeInstrumentation.getAttributesProperty;
         this.upload                     = this.sandbox.upload.upload;
-        this.get                        = require;
+
+        // Used in client tests
```

Comment:
Wrong. It's used in tests, we need to describe how it used as well. Discuss text with @VasilyStrelyaev